src/loaders/common/loader_helper.py

```diff
@@ -184,6 +184,69 @@ def is_upa_info_complete(upa_dir: str):
     return True
 
 
+def make_collection_source_dir(
+        root_dir: str,
+        collection_source_dir: str,
+        env: str,
+        collection: str,
+        release_ver: str
+) -> str:
+    """
+    Helper function that creates a collection & source_version and link in data
+    to that colleciton from the overall source data dir.
+    """
+    csd = os.path.join(root_dir, collection_source_dir, env, collection, release_ver)
+    os.makedirs(csd, exist_ok=True)
+    return csd
+
+
+def create_softlinks_in_csd(csd: str, work_dir: str, genome_ids: list[str], taxonomy_files: list[str] = []) -> None:
```

Comment:
avoid mutable defaults. 

```suggestion
def create_softlinks_in_csd(csd: str, work_dir: str, genome_ids: list[str], taxonomy_files: list[str] = []) -> None:
```

    if taxonomy_files is None:
        taxonomy_files = []

Reply:
Updated!